operator/src/main/java/com/oracle/coherence/k8s/operator/CoherenceOperator.java

```diff
@@ -153,13 +153,17 @@ static void assertNamespaces(String[] asIncludedNamespaces, String[] asExcludedN
          */
         NamespaceProcessor(String sNamespace, String[] asIncludedNamespaces, String[] asExcludedNamespaces)
             {
-            m_sElasticsearchHost = ((System.getenv(ES_HOST) == null) ?
+            m_sElasticsearchHost     = ((System.getenv(ES_HOST) == null) ?
                     "elasticsearch." + sNamespace + ".svc.cluster.local" : System.getenv(ES_HOST));
 
-            m_sElasticsearchPort = ((System.getenv(ES_PORT) == null) ? DEFAULT_ES_PORT :
+            m_sElasticsearchPort     = ((System.getenv(ES_PORT) == null) ? DEFAULT_ES_PORT :
                     Integer.toString(Integer.parseInt(System.getenv(ES_PORT))));
 
-            m_sOperatorHost      = "coherence-operator-service." + sNamespace + ".svc.cluster.local";
+            m_sElasticsearchUser     = ((System.getenv(ES_USER) == null) ? "" : System.getenv(ES_USER));
+
+            m_sElasticsearchPassword = ((System.getenv(ES_PASSWORD) == null) ? "" : System.getenv(ES_PASSWORD));
+
+            m_sOperatorHost          = "coherence-operator-service." + sNamespace + ".svc.cluster.local";
```

Comment:
This code is so repetitive, it could definitely use some cleanup.

At the very least, ternary operators could be replaced by:

```Optional.ofNullable(System.getenv(VAR)).orElse(DEFAULT);```

Better yet, we can add a simple class `Env`, that simplifies dealing with env variables:

```
public class Env {
    public static String get(String name, String defaultValue) {
        return Optional.ofNullable(System.getenv(name)).orElse(defaultValue);
    }
}
```
in which case the above simply end up being:
```
m_sElasticsearchHost = Env.get(ES_HOST, "elasticsearch." + sNamespace + ".svc.cluster.local");
m_sElasticsearchPort = Env.get(ES_PORT, DEFAULT_ES_PORT);
// etc.
```

Reply:
I created #203 to address this as a separate issue.